includes/HttpConfig.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef HTTPCONFIG_HPP
+# define HTTPCONFIG_HPP
+
+# include <string>
+
+# include "HttpConfigServer"
+
+class HttpConfig
+{
+	private:
+		std::string m_include;
+		std::string m_root;
+		vector<HttpConfigServer> m_httpServer();
+	private:
+		HttpConfig();
+		virtual ~HttpConfig();
+		HttpConfig(const HttpConfig &other);
```

Comment:
```suggestion
		HttpConfig(HttpConfig const &other);
```
@jwon42 님이 말해주신 const 위치차이는 다시 한번더 찾아보겠습니다.!

Reply:
우선 [여기](https://www.quora.com/What-are-the-differences-between-int-const-int-const-const-int-and-const-int-in-C)에서 말하기로는 둘이 같은 의미라고 토론이있었습니다.  

다른점을 못 찾겠어서 혹시 알려주실 수 있나요?

우선 코딩 컨벤션은 ```HttpConfig(HttpConfig const &other);```이걸로 다 고치겠습니다!

>``` int const&``` and ```const int&``` mean same thing - a reference to an int or const int object. The target object cannot be modified through such reference.

- ```int& const``` X
  - a constant (const) reference (&) to int. Since references refer to the same object since their creation, they are always constant. The const is redundant. __Some compilers warn about it, other emit errors.__
  - const 중복됨.
- ```int const&``` O
  -  reference (&) to a constant (const) int. The integer can't be modified through the reference.  
- ```const& int``` X
  - const 참조에 대한 정수. 말이되지 않는 문법.
- ```const int&``` O
  - const 인 int 객체에 대한 참조. ```int const &```와 동등하다.

- - -
## + 포인터 복습

- ```const int* ptr```
- ```int* const ptr```
- ```const int* const ptr```

1. ```const int* ptr```
```c++
#include <iostream>
 
using namespace std;
 
int main()
{
    int value1 = 5;
    const int* ptr = &value1;
 
    int value2 = 6;
    ptr = &value2;
     // *ptr = 6; 불가능
}
```

2. ```int* const ptr```
```c++
#include <iostream>
 
using namespace std;
 
int main()
{
    int value1 = 5;
    int* const ptr = &value1;
 
    int value2 = 6;
    //ptr = &value2; 불가능
     *ptr = 6;
}
```
3. ```const int* const ptr```
```c++
#include <iostream>
 
using namespace std;
 
int main()
{
    int value1 = 5;
    const int* const ptr = &value1;
 
    int value2 = 6;
    // *ptr = 6; 불가능
    // ptr = &value2; 불가능
}
```

#### 참고 링크
- https://spenshi.tistory.com/entry/%ED%8F%AC%EC%9D%B8%ED%84%B0%EC%99%80-const
- [What does 'const&' mean in C++? [closed]](https://stackoverflow.com/questions/19415674/what-does-const-mean-in-c)
- http://www.cplusplus.com/forum/beginner/149060/
- https://www.quora.com/What-are-the-differences-between-int-const-int-const-const-int-and-const-int-in-C